src/Microsoft.Dnx.Runtime/PathSearchBasedProjectResolver.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Dnx.Runtime
+{
+    public class PathSearchBasedProjectResolver : IProjectResolver
```

Comment:
Why is it still in the runtime? Isn't the runtime only using the LockFile table now?